node/tests/RedisClient.test.ts

```diff
@@ -148,10 +148,21 @@ describe("GlideClient", () => {
                 setTimeout(resolve, 500);
             });
 
+            const blmpopPromise = client.blmpop(
+                ["key1", "key2"],
+                ListDirection.LEFT,
+                0.1,
+            );
+            const timeoutPromise2 = new Promise((resolve) => {
+                setTimeout(resolve, 500);
+            });
+
             try {
                 await Promise.race([blmovePromise, timeoutPromise]);
+                await Promise.race([blmpopPromise, timeoutPromise2]);
```

Comment:
Can we do this? And can we also add other blocking commands to this test.

```suggestion
                await Promise.race([blmovePromise, blmpopPromise, timeoutPromise]);
```